CHANGELOG.md

```diff
@@ -45,7 +45,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Initial release of cisco-cookbook for chef, supporting Cisco NX-OS software release 7.0(3)I2(1) on Cisco Nexus switch platforms: N95xx, N93xx, N30xx and N31xx.
 - Please note: 0.9.0 is an EFT pre-release for a limited audience with access to NX-OS 7.0(3)I2(1). Additional code changes may occur in 0.9.x prior to the final 1.0.0 release.
 
-[unreleased]: https://github.com/cisco/cisco-network-chef-cookbook/compare/master...develop
```

Comment:
I did it here because we took it out for puppet and then added it back post-release. But... you're absolutely right, we could have just left it. :-)